tests/benchmarks/benchmark_runner.py

```diff
@@ -24,8 +26,82 @@ def retries(request):
     return int(request.config.getoption("--retries"))
 
 
+async def grade(to_grade, prompt):
```

Comment:
I got this error testing locally on the Clojure one:
<img width="893" alt="image" src="https://github.com/AbanteAI/mentat/assets/50287275/9e6e84d2-f303-44e8-9144-4a6abbe181dc">

Maybe here we should check/truncate before sending to gpt4.

Reply:
Related - might be a good idea to put a try/except block around each individual test, so if one of them fails, you still get the results of the rest printed to the screen.